src/slicing/static/static-slicer.ts

```diff
@@ -33,7 +32,7 @@ export function staticSlicing(
 	graph: DataflowGraph,
 	{ idMap }: NormalizedAst,
 	criteria: SlicingCriteria,
-	threshold = getConfig().solver.slicer?.threshold ?? 75,
+	threshold = 75,
```

Comment:
We should definitely not lose the configurable threshold! 
Let's update the parameter to have an undefined default, or accept the config explicitly